### ISSUE TITLE
Environment variable does get resolves in 'VM Options' in 'Configuration'

### ISSUE THREAD

**Issue by dhruv-bansal** [Ashald/EnvFile#84]
Defined variable in file 
`DATASOURCE_CONF=-Dspring.datasource.url=jdbc:mysql://localhost:3306/TEST_DATABASE -Dspring.datasource.username=test -Dspring.datasource.password=test -Dhibernate.dialect=org.hibernate.dialect.MySQLDialect`

added file in configuration 
![image](https://user-images.githubusercontent.com/11644608/59099773-c3a8a600-8941-11e9-8d9f-4454d3be8aba.png)

This variable is not getting resolved in VM Options
![image](https://user-images.githubusercontent.com/11644608/59099640-4d0ba880-8941-11e9-9174-c06881a24e46.png)

**Comment by ashald** [comment]
Does it work if you set environment variables via native IDE UI in the `Configuration` tab?

I'm afraid it might be a limitation that we cannot do anything about in the same way as it was in https://github.com/ashald/EnvFile/issues/78#issuecomment-486115324.

If I had to offer an alternative, I'd suggest trying setting those values via `JAVA_OPTS` environment variable.

**Comment by dhruv-bansal** [comment]
Yes, it worked very well after declaring into "Environment variables" in the "Configuration tab".
I just have to change ${DATASOURCE_CONF} to $DATASOURCE_CONF$
In my point of view, EnvFile should act as a Decorator over the "Environment Variables" functionality.  Everything offered by "Environment variables" plus the additional feature of easily managing variable via file etc.
@ashald What do you think?

**Comment by ashald** [comment]
I think that's a great idea but unfortunately that is not something that is possible to implement with current APIs exposed by JetBrains IDEs. 😞
